src/Command/CommandHelper.php

```diff
@@ -199,7 +208,15 @@ public static function begin(
 			} catch (InvalidStateException | FileNotFoundException $e) {
 				$errorOutput->writeLineFormatted($e->getMessage());
 				throw new InceptionNotSuccessfulException();
+			} catch (TypeError $e) {
+				if (pathinfo($projectConfigFile, PATHINFO_EXTENSION) === 'php') {
+					$errorOutput->writeLineFormatted(sprintf('Invalid .php project config given: ' . $e->getMessage()));
```

Comment:
I don't like this change. A TypeError can happen for multiple reasons. It might be a bug in PHPStan itself, and you'd give the user a very misleading error message (and also throwing away the stack trace).